src/Orleans.Core/Configuration/OptionLogger/OptionFormatterExtensionMethods.cs

```diff
@@ -51,7 +55,7 @@ public static class OptionConfigureExtensionMethods
             where TOptionFormatterResolver : class, IOptionFormatterResolver<TOptions>
         {
             services.AddSingleton<IOptionFormatterResolver<TOptions>, TOptionFormatterResolver>()
-                .AddSingleton<IOptionFormatter>(sp => sp.GetService<IOptionFormatterResolver<TOptions>>().Resolve(name));
+                    .AddSingleton<IOptionFormatter>(sp => sp.GetService<IOptionFormatterResolver<TOptions>>().Resolve(name));
```

Comment:
if some user called 
```
ConfigureFormatter<TOptions, TOptionFormatterResolver1>(name1);
ConfigureFormatter<TOptions, TOptionFormatterResolver2>(name1);
```
Will this result in duplicated IOptionFormatter for name1 option in DI? I didn't see a test case covering this case